config/routes.rb

```diff
@@ -1,5 +1,9 @@
 Rails.application.routes.draw do
+  get 'static_pages/home'
```

Comment:
I don't have too many opinions on this yet, but isn't it kind of weird that you'd go to `bravo.com/static_pages/home` rather than `bravo.com/home`? Initially, this is fine, since you can change the paths.

One way to solve this is to change the routing to something like `get '/home', to: 'static_pages#home'`